lib/field-locale.ts

```diff
@@ -82,10 +85,18 @@ export default class FieldLocale implements FieldAPI {
     return this._valueSignal.attach(handler)
   }
 
+  getIsDisabled(): boolean {
+    return this._isDisabledSignal.getMemoizedArgs()[0]
```

Comment:
Will this always be the same value? i.e: always called with initial value

Reply:
No, `getMemoizedArgs()` always returns the most recent arguments. The reason for `[0]` is that the values are stored as an array, but we only care about the first.

Reply:
Ah - we're prepending instead of appending the last values

Reply:
No, checkout the implementation of MemoizedSignal: https://github.com/contentful/ui-extensions-sdk/blob/master/lib/signal.ts#L24. The signal always only keeps the last value. But the value is always an array because it expects to store the list of arguments of a function call. In our case we only care about the first argument of the function call.